### ISSUE TITLE
Exported log isn't imported properly

### ISSUE THREAD

**Issue by erazemk** [lainsce/khronos#115]
I'm using Khronos 3.7.0 from Flathub.

When I try exporting a log, it works fine, but if I try to import that same log, it shows the start and finish time as null.
This is the exported log:
```csv
task,timedate,tags
"Test","00∶00∶05sob, 25/06 16∶05∶42 – 16∶05∶48","Tag"
```

My guess is this is due to improperly parsing the date, which in my case is in the Slovenian locale.
I'm also guessing this issue can be resolved by forcing a specific locale for exports, instead of using the system-defined one (`2022-06-25 16:05:42-16:05:48` seems much more readable in exports to me anyway).

When imported the result looks like this:
![image](https://user-images.githubusercontent.com/46302511/175777252-b50c425e-64bf-4b10-9116-2f18c503b713.png)

**Comment by lainsce** [comment]
Yeah, the export/importing expects english as the language, and to force it on export, I'd have to research this.

**Comment by jayvogt** [comment]
Just wanted to add that this still seems to be an issue, even with english as my language:
```
task,timedate,tags
"Test log","00∶06∶50Tue, 17/10 09∶54∶40 – 10∶01∶36","testing:planning:new app"
"Testing log 2","01∶35∶51Tue, 17/10 10∶02∶45 – 11∶38∶31","testing:planning:new app"
```
My guess would have been that there's no separation between the timer (`01:35:51`) and the day/date (`Tue, 17/10`). I tried updating my import to `"Testing log 2", "01:35:51", "Tue, 17/10 10:02:45 – 11∶38∶31","testing:planning:new app"`, but it just crashes the app.

**Comment by 0Volcanon** [comment]
I found that there are two problems here.
1. How the log is exported, the Logs will work if you add a space between the time and day at the first set inside timedate.
2. The date is using the wrong format. It's failing because it expects a month first, day later. So, switching the dates around works. I don't think this is a problem at export but rather how the program is expecting the dates to be.
At least I think this is what's happening, I don't know Vala. But this does make the logs work.
![Doesn't Work](https://github.com/user-attachments/assets/ddf4108d-358d-4860-8625-359aa6fd47fa)
![Works](https://github.com/user-attachments/assets/8a444228-eefb-4db2-bca2-d2d69faf6e10)
